refresh_test.py

```diff
@@ -13,46 +13,53 @@
 #
 # Copyright (c) 2017 ScyllaDB
 
-import time
+import random
 
 from avocado import main
 from sdcm.tester import ClusterTester
-from sdcm.nemesis import RefreshMonkey
-from sdcm.nemesis import RefreshBigMonkey
+from sdcm.utils import remote_get_file
 
 
 class RefreshTest(ClusterTester):
     """
     Nodetool refresh after uploading lot of data to a cluster with running load in the background.
     :avocado: enable
     """
-    def test_refresh_small_node(self):
-        self.db_cluster.add_nemesis(nemesis=RefreshMonkey,
-                                    loaders=self.loaders,
-                                    monitoring_set=self.monitors)
+    def get_random_target(self):
+        return random.choice(self.db_cluster.nodes)
 
-        # run a write workload
-        stress_queue = self.run_stress_thread(stress_cmd=self.params.get('stress_cmd'),
-                                              stress_num=2,
-                                              keyspace_num=1)
-        time.sleep(30)
-        self.db_cluster.start_nemesis()
-        self.db_cluster.stop_nemesis(timeout=None)
+    def prepare_sstable(self, node):
```

Comment:
Right it support to download from S3 or copy from the GCE image.
Uncompressing a 72G tar.gz takes about 20+ mins.